app/services/analytics_events.rb

```diff
@@ -500,6 +500,42 @@ def idv_verify_in_person_troubleshooting_option_clicked
     track_event('IdV: verify in person troubleshooting option clicked')
   end
 
+  # The user visited the in person proofing location step
+  # @param [String] flow_path Path used for document capture ("standard" or "hybrid")
+  def idv_in_person_location_visited(**extra)
```

Comment:
These are parameters we expect, but we can't list them as keyword arguments, due to the issue previously discovered in #6791.

Separately, I've been wondering if we could construct keyword arguments by inspecting the analytics method and picking the parameters from the request.

Pseudo-code:

```rb
payload.with_indifferent_access.slice(
  *analytics_method.parameters.select { |info_pair| info_pair[0] == :keyreq }.map(&:last),
)
```

Reply:
The linter didn't like the unused `flow_path` parameter to be documented, so I had to remove it in afd4dc45a.

Reply:
😬 the linter was trying to do good work.. I would be a fan of that string -> symbol dynamic construction of params if we think we could get it to work

Reply:
> > 😬 the linter was trying to do good work.. I would be a fan of that string -> symbol dynamic construction of params if we think we could get it to work
> 
> I'll plan to revisit that separately.

See: #6927